apps/api/src/server/appeals/appeals/appeals.formatter.js

```diff
@@ -82,7 +87,20 @@ const appealFormatter = {
 			siteVisit: {
 				visitDate: appeal.siteVisit?.visitDate || null
 			},
-			startedAt: appeal.startedAt
+			startedAt: appeal.startedAt,
+			documentationSummary: {
```

Comment:
Can you please update Swagger to display this new object in the example response?

The example response is `SingleAppealResponse` in `apps/api/src/server/swagger.js`

Reply:
I'd missed this, thanks - updated